### PR TITLE
Update docs of Ballerina track

### PR DESCRIPTION
**Pull request by praneesha**
Update docs of Ballerina track

### CODE REVIEW

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
 # Installation
 
-You can install Ballerina by following the Ballerina [installation guide][install].
-
-[install]: https://ballerina.io/learn/install-ballerina/set-up-ballerina/
+To install Ballerina, set the development environment up, and write a simple Ballerina program, go to [Get started](https://ballerina.io/learn/get-started/).
```
Comment (anupama-pathirage) [inline docs/INSTALLATION.md:3]: Add a line on version compatibility too

File: docs/LEARNING.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Learning
 
-Ballerina [learn](https://ballerina.io/learn) page contains multiple resources you to get up to speed with the language, master the concepts, and build comprehensive real-world use cases. 
\ No newline at end of file
+The Ballerina [learn](https://ballerina.io/learn) section contains multiple resources for you to get up to speed with the language, master the concepts, and build comprehensive real-world use cases.
```
Comment (anupama-pathirage) [inline docs/LEARNING.md:3]: Page is better than section.

Comment (anupama-pathirage) [reply docs/LEARNING.md:3]: Also add a list of links for key resources.

Comment (praneesha) [reply docs/LEARNING.md:3]: We have the resources listed in the `RESOURCES.md `file. Will add a link to it.

File: docs/RESOURCES.md [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
-- [StackOverflow][stackoverflow]
-- Exercism's resources
-    - get help on [the forum][exercism-forum], or
-    - in the #support channel on [Exercism's discord][exercism-discord].
+- Ballerina official website
+    - [Get Started](https://ballerina.io/learn/get-started/)
+    - [Ballerina By Example](https://ballerina.io/learn/by-example/)
+    - [Learn Ballerina](https://ballerina.io/learn/)
+    - [Ballerina specifications](https://ballerina.io/learn/ballerina-specifications/)
+    - [Community](https://ballerina.io/community/)
```
Comment (anupama-pathirage) [inline docs/RESOURCES.md:10]: No need of community link here.

File: docs/RESOURCES.md [hunk 4]
```diff
@@ -1,20 +1,19 @@
...
 
-- the Ballerina web site
-    - [Getting Started][getting-started]
-    - [Ballerina Examples][ballerina-examples]
-    - [Join Discord][join-discord]
-- [StackOverflow][stackoverflow]
-- Exercism's resources
-    - get help on [the forum][exercism-forum], or
-    - in the #support channel on [Exercism's discord][exercism-discord].
+- Ballerina official website
```
Comment (anupama-pathirage) [inline docs/RESOURCES.md:5]: Instead of official website, say something like "Ballerina learning resources"

File: docs/INSTALLATION.md (hunk 1 above)
Comment (praneesha) [reply docs/INSTALLATION.md:3]: Added.

File: docs/LEARNING.md (hunk 2 above)
Comment (praneesha) [reply docs/LEARNING.md]: Done.

File: docs/RESOURCES.md (hunk 3 above)
Comment (praneesha) [reply docs/RESOURCES.md:10]: Removed.

File: docs/RESOURCES.md (hunk 4 above)
Comment (praneesha) [reply docs/RESOURCES.md:5]: Changed.

File: docs/RESOURCES.md [hunk 5]
```diff
@@ -1,20 +1,19 @@
...
+    - [Get Started](https://ballerina.io/learn/get-started/)
+    - [Ballerina By Example](https://ballerina.io/learn/by-example/)
+    - [Learn Ballerina](https://ballerina.io/learn/)
+    - [Ballerina specifications](https://ballerina.io/learn/ballerina-specifications/)
+    - [Community](https://ballerina.io/community/)
+- Other documentation resources
```
[the remark this answers is not in the document]
Comment (praneesha) [reply docs/RESOURCES.md:11]: Done.

File: docs/RESOURCES.md [hunk 6]
```diff
@@ -1,20 +1,12 @@
 # Resources
 
-To get help if you're having trouble, you can use one of the following resources:
+- Learning resources
+    - [Get Started](https://ballerina.io/learn/get-started/)
+    - [Ballerina By Example](https://ballerina.io/learn/by-example/)
+    - [Learn Ballerina](https://ballerina.io/learn/)
```
Comment (anupama-pathirage) [inline docs/RESOURCES.md:6]: Add this as last one as it is the page which contains all the others as well.

File: docs/RESOURCES.md [hunk 7]
```diff
@@ -1,20 +1,12 @@
...
+    - [Ballerina specifications](https://ballerina.io/learn/ballerina-specifications/)
+    - [Ballerina API Docs](https://lib.ballerina.io/)
+- Community channels
```
Comment (anupama-pathirage) [inline docs/RESOURCES.md:9]: Shall we add a section for Twitter and LinkedIn pages too.

File: docs/INSTALLATION.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 # Installation
 
-You can install Ballerina by following the Ballerina [installation guide][install].
+To install Ballerina, set the development environment up, and write a simple Ballerina program, go to [Get started](https://ballerina.io/learn/get-started/).
```
[the remark this answers is not in the document]
Comment (praneesha) [reply docs/INSTALLATION.md:3]: Updated.

File: docs/RESOURCES.md (hunk 6 above)
Comment (praneesha) [reply docs/RESOURCES.md:6]: Updated.

File: docs/RESOURCES.md (hunk 7 above)
Comment (praneesha) [reply docs/RESOURCES.md:9]: Added.

**Comment by anupama-pathirage** [comment]
Looks good.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**